contracts/instance/base/ObjectLifecycle.sol

```diff
@@ -57,6 +58,10 @@ contract ObjectLifecycle is
         setStateTransition(POLICY(), COLLATERALIZED(), CLOSED());
     }
 
+    function _setupPremiumLifecycle() private {
```

Comment:
should we consider to move the premium info from policyInfo to premiumInfo and have 2 states, like active and paid?

Reply:
solved with #535